main.go

```diff
@@ -112,9 +112,14 @@ func main() {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt)
 	go func() {
-		sig := <-c
-		logger.Warnf("Got signal %s, unmounting", sig)
-		server.Unmount()
+		for {
+			sig := <-c
+			logger.Warnf("Got signal %s, unmounting", sig)
+			err := server.Unmount()
+			if err != nil {
+				logger.Warnf("Error while unmounting: %v", err)
```

Comment:
If there's an error, we don't exit and wait for another signal? I think rather we should have a loop that tries to exit until it succeeds, maybe with a timeout.